release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -414,8 +495,6 @@ inspect them using normal `oc` CLI commands.
 move to RPMs or the kubelet system containers (by setting
 `openshift_use_system_containers=true`), to start or run the platform.
 
-* `openshift_node_bootstrap` is now removed. All nodes must be bootstrapped.
-
 * openshift-sdn and openvswitch, if enabled, will be run in a daemonset.
```

Comment:
I can't understand what does it mean "if enabled" (I might be wrong to get it). In 3.10, openshift-sdn and openvswitch will definitely be run as daemonset, no option to disable or enable.

Reply:
@ganhuang Thanks! This is also addressed in https://github.com/openshift/openshift-docs/pull/11106